src/bundle/Command/CronRunCommand.php

```diff
@@ -46,6 +47,30 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $cron->setExecutor(new Executor());
         $cron->setResolver($resolver);
 
-        $cron->run();
+        $reports = $cron->run();
+
+        while ($cron->isRunning()) {}
+
+        if($this->getContainer()->has('monolog.logger.cronjob')){
+            $logger = $this->getContainer()->get('monolog.logger.cronjob');
+            if($logger){
+                foreach ($reports->getReports() as $report) {
+                    $extraInfo = array(
+                        'command' => $report->getJob()->getProcess()->getCommandLine(),
+                        'exitCode' => $report->getJob()->getProcess()->getExitCode()
+                    );
+                    foreach($report->getOutput() as $reportOutput){
+                        $logger->info($reportOutput, $extraInfo);
+                    }
+                    if(!$report->isSuccessful()){
+                        foreach($report->getError() as $reportError){
+                            if(!empty(trim($reportError))){
+                                $logger->error(trim($reportError), $extraInfo);
+                            }
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
I'm sorry, but we cannot accept this in such form. It's not going to work in eZ Platform 3.x / Symfony 5 and at this point it's a technical debt.
1. You cannot use `get` on container, since the services are or should be private for the most cases. You need to inject Logger into the constructor.
2. You need to use abstraction (LoggerInterface) instead of Monolog. While here for Symfony 3.4 specific instance needs to be injected, on upper branch it can be [autowired by parameter name](https://symfony.com/doc/current/logging/channels_handlers.html#how-to-autowire-logger-channels). Regardless of that LoggerInterface needs to be used.

Reply:
@alongosz Hope the latestet commit fixes the issues